man/memkind_memtier.3

```diff
@@ -16,15 +16,11 @@ functionality is considered as stable API (STANDARD API).
 .sp
 .B "TIER MANAGEMENT:"
 .sp
-.BI "struct memtier_tier *memtier_tier_new(memkind_t " "kind" );
```

Comment:
Remove this as well https://github.com/memkind/memkind/blob/feat-memory-tiering/man/memtier.7#L17